### DISCUSSION TITLE
Meaning and Examples of Business Logic as it Pertains to Applications

### DISCUSSION THREAD

**Discussion by Benanna2019** [hyper63/atlas#76] [Q&A] [answered]
When we use the word/phrase business logic, what are we referring too? Maybe a list of examples of business logic? Are we making, api calls, authentication, etc?

**Comment by twilson63** [comment, marked as answer]
Business Logic is the rules and algorithms that make your app special. Does a css framework make your app special? Does a database, search engine, cache?

For example, let’s take an app that does real time golf scoring for a golf tournament. So this app, needs a ui, a design, a database, a way to manage distribute notifications. These are general logic requirements.

recording your golf score per hole, displaying each players score real time in a leader board, are specific logic requirements that is special to your app.

this is what is referred to as business logic. Some may refer to it as functional requirements
